src/load-services.ts

```diff
@@ -0,0 +1,19 @@
+import express, { Application } from 'express';
+import AccesstokenServiceManager from './modules/access-token/access-token-manager';
+import AccountServiceManager from './modules/account/account-service-manager';
+
+export default class MicroServices {
```

Comment:
Let's create a private method in `app.ts` - `createRestAPIServer`. That method would do what's being done in this method.